lineoperations/src/lo_fns.c

```diff
@@ -20,22 +20,40 @@
 */
 
 
-#include "linefunctions.h"
+#include "lo_fns.h"
+#include "lo_prefs.h"
+
+
+/* Get sort function based on user preferences */
+lo_strcmpfns
+getcmpfns(void)
+{
+	if(lo_info->use_collation_compare)
+	{
+		return g_utf8_collate;
+	}
+	else
+	{
+		return strcmp;
```

Comment:
Maybe you like to prefer ```g_strcmp0``` from the glib here? The same as ```strcmp``` just a bit safer but from glib.

Reply:
For some reason I changed it a few years ago https://github.com/geany/geany-plugins/commit/2e2e043318fd2630cd05f1ee95c49fcdb019b6f7, I just don't remember why.

I agree with you, I think it is better with glib. I tested it and do not see any problems, so I'm not sure what I was thinking a few years ago.